ui/app/styles/components/table.scss

```diff
@@ -55,7 +55,22 @@
     }
   }
 
+  .w-1\/2 {
+    width: 50%;
+  }
+  .w-1\/3 {
+    width: 33.333%;
+  }
+  .w-1\/4 {
+    width: 25%;
+  }
+  .w-1\/5 {
+    width: 20%;
+  }
   .w-2\/5 {
     width: 40%;
   }
+  .w-3\/5 {
+    width: 60%;
+  }
```

Comment:
Let me know what y’all think of these helper classes. Can add more documentation, or do them a different way, I’m not precious.

The idea is that you’d use them on the `<col>` element, as per the example.